scripts/bench/bench_local.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -e
+
+# Benchmark with a local version
+# Usage: ./scripts/bench/bench_local.sh gosec v1.59.0
+
+# ex: gosec
+LINTER=$1
+
+# ex: v1.59.0
+VERSION=$2
```

Comment:
Maybe we should add a check that `LINTER` and `VERSION` are set and non-empty.

Reply:
it's not really useful because the script is only used by the make target, i.e. if one element is missing the other element will be used and so this will create confusion.

ex: if you forget the linter, the version will be at `$1` and not `$2`.